### DISCUSSION TITLE
Install - No MAC

### DISCUSSION THREAD

**Discussion by Amrani-IT** [SoCuul/SCInsta#87] [Support] [answered]
Is it possible to do this without a mac

**Comment by SoCuul** [comment]
It should be possible to build and bundle this tweak on linux, as if I remember correctly @asdfzxcvbn was doing this.

**Comment by asdfzxcvbn** [comment, marked as answer]
> It should be possible to build and bundle this tweak on linux, as if I remember correctly @asdfzxcvbn was doing this.

yep works fine through linux, just needs a modification to flex to build it
